src/python/Utils/procstats.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+#pylint: disable=
```

Comment:
Alan, I rather like to keep it since it allows to run this as standalone script and does not affect anything else.